src/sketchup-stl.rb

```diff
@@ -52,7 +52,7 @@ def self.translate(string)
       'This is an open source project sponsored by the SketchUp team. More ' <<
       'info and updates at https://github.com/SketchUp/sketchup-stl'
     )
-    extension.version = '2.1.6'
```

Comment:
Hm.. maybe? The squash-merge is a rather new feature of GitHub. But it allow us to merge a Pull Request with many small follow-up commits into a single one. Makes the history cleaner and easier to read.